imports/collections/schemas/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Reaction uses {@link https://github.com/aldeed/simple-schema-js SimpleSchema} to apply basic content and structure validation to Collections.
+ * See {@link https://docs.reactioncommerce.com/reaction-docs/master/simple-schema full documentation}.
+ * @typedef {SimpleSchema} SimpleSchema
+ * @summary SimpleSchema for Collections
+ * @namespace Schemas
+ */
+
+/**
+ * Reaction uses {@link https://github.com/aldeed/simple-schema-js SimpleSchema} to apply basic content and structure validation to Collections.
```

Comment:
Do we need both of these jsdoc blocks?

Reply:
I think so, I believe @aldeed  did this when he cleaned up the jsdoc a few months back